releases/release-1.31/release-notes/maps/pr-123233-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 123233
+releasenote:
+  text: Ensured daemonset controller counts old unhealthy pods towards max unavailable
```

Comment:
```suggestion
  text: Ensured daemonset controller counts old unhealthy pods towards max unavailable.
```

Reply:
@npolshakova the last word of the sentence is "budget" which is missing the period :D